package/netflix5/netflix5.mk

```diff
@@ -224,6 +234,12 @@ NETFLIX5_POST_EXTRACT_HOOKS += NETFLIX5_FIX_CONFIG_XMLS
 
 ifeq ($(BR2_PACKAGE_NETFLIX5_LIB),y)
 
+ifeq ($(BR2_PACKAGE_WPEFRAMEWORK_COMPOSITOR),y)
```

Comment:
Does that mean if compositor is not set the NETFLIX5_INSTALL_WPEFRAMEWORK_XML does not get defined? Wont this throw an error if compositor is turned off?

Reply:
This is defining the macro for copying NETFLIX5_INSTALL_WPEFRAMEWORK_XML
While turning off, this will be empty and will not make issue

Reply:
You're right! My bad :)